server/src/data/db/connection.js

```diff
@@ -1,6 +1,6 @@
-import Sequelize from 'sequelize';
```

Comment:
let's remove `connection.js` and do all this logic in the index file
ex: https://github.com/BinaryStudioAcademy/bsa-2021-jabber/blob/development/backend/src/server.ts#L19